specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json

```diff
@@ -1472,6 +1472,75 @@
         "entryFilePath"
       ]
     },
+    "ExecuteSSISPackageActivity": {
+      "description": "Execute SSIS package activity.",
+      "x-ms-discriminator-value": "ExecuteSSISPackage",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ExecutionActivity"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "x-ms-client-flatten": true,
+          "description": "Execute SSIS package activity properties.",
+          "$ref": "#/definitions/ExecuteSSISPackageActivityTypeProperties"
+        }
+      },
+      "required": [
+        "typeProperties"
+      ]
+    },
+    "ExecuteSSISPackageActivityTypeProperties": {
+      "description": "Execute SSIS package activity properties.",
+      "properties": {
+        "packageLocation": {
+          "description": "SSIS package location.",
+          "$ref": "#/definitions/SSISPackageLocation"
+        },
+        "use32bitRuntime": {
+          "description": "Specifies whether to use 32 bit runtime to execute SSIS package.",
+          "type": "string",
+          "enum": [
+            "Yes",
+            "No"
+          ],
+          "x-ms-enum": {
+            "name": "SSISExecutionUse32bitRuntime",
+            "modelAsString": true
+          }
+        },
+        "loggingLevel": {

```

Comment:
This looks like something that should be an enum, or referencing a previously declared enum to me.